packages/riipen-ui-docs/src/modules/components/MarkdownPage.jsx

```diff
@@ -11,141 +18,164 @@ import Menu from "src/modules/components/Menu";
-
-    const theme = this.context;
-
-    const demos = {};
-    let markdown;
-
-    req.keys().forEach(filename => {
-      if (filename.indexOf(".md") !== -1) {
-        markdown = req(filename).default;
-      } else if (filename.indexOf(".jsx") !== -1) {
-        const demoName = `${path}/${filename
-          .replace(/\.\//g, "")
-          .replace(/\.jsx/g, ".js")}`;
-
-        demos[demoName] = {
-          name: demoName,
-          jsx: req(filename).default,
-          rawJS: reqSource(filename).default
-        };
-      }
-    });
-
-    const sections = getSections(markdown);
-
-    return (
-      <div>
-        <Head>
-          <title>{title ? `${title} | ` : ""}Riipen UI</title>
-        </Head>
-        <AppBar>
-          <Link href="/" color="inherit">
-            Riipen-UI
-          </Link>
-        </AppBar>
-        <div className="page">
-          <div className="menu">
-            <Menu />
-          </div>
-          <div className="content">
-            {sections.map((section, index) => {
-              if (demos && constants.REGEX.DEMO.test(section)) {
-                let options;
-
-                try {
-                  options = JSON.parse(`{${section}}`);
-                } catch (err) {
-                  console.error(err);
-                  return null;
-                }
-
-                const name = options.demo;
-
-                if (!demos || !demos[name]) {
-                  throw new Error(
-                    [
-                      `Missing demo: ${name}. You can use one of the following:`,
-                      Object.keys(demos)
-                    ].join("\n")
-                  );
-                }
-
-                return (
-                  <Demo key={index} demo={demos[name]} options={options} />
+  const demos = {};
+  let markdown;
+
+  req.keys().forEach((filename) => {
+    if (filename.indexOf(".md") !== -1) {
+      markdown = req(filename).default;
+    } else if (filename.indexOf(".jsx") !== -1) {
+      const demoName = `${path}/${filename
+        .replace(/\.\//g, "")
+        .replace(/\.jsx/g, ".js")}`;
+
+      demos[demoName] = {
+        name: demoName,
+        jsx: req(filename).default,
+        rawJS: reqSource(filename).default
+      };
+    }
+  });
+
+  const sections = getSections(markdown);
+
+  return (
+    <div>
+      <Head>
+        <title>{title ? `${title} | ` : ""}Riipen UI</title>
+      </Head>
+      <AppBar>
+        <Link href="/" color="inherit">
+          Riipen-UI
+        </Link>
+        <div className="menuMobile">
+          <React.Fragment>
+            <ButtonIcon color="white" onClick={handleMenuOpen}>
+              <FontAwesomeIcon icon={faBars} />
+            </ButtonIcon>
+            <Popover
+              anchorEl={anchorState}
+              closeOnScrolled={false}
+              fullWidth
+              isOpen={menuOpen}
+              lockScroll
+            >
+              <Menu />
```

Comment:
Lazily just threw current Menu into a Popover... Could use Menu/MenuItem potentially 😅

Reply:
Nice! I tried this out in the UI, but there seems to be no way to actually close the menu.

![Screenshot from 2021-06-04 08-18-53](https://user-images.githubusercontent.com/1061076/120824900-b82aee00-c50d-11eb-90fd-b8a1bd504c30.png)